mne/viz/_3d.py

```diff
@@ -984,6 +1077,10 @@ def plot_source_estimates(stc, subject=None, surface='inflated', hemi='lh',
     SUBJECTS_DIR environment variable or always use the same value for
     subjects_dir (within the same Python session).
 
+    This function uses Mayavi to plot the source estimates. If Mayavi is not
+    installed, the plotting is done with matplotlib (extremely slow,
+    interactivity disabled).
```

Comment:
Thinking ahead, I think we should add a `backend='auto' (PR behavior) | 'mayavi' | 'mpl'` kwarg for this.